cmake/Macros.cmake

```diff
@@ -52,6 +55,11 @@ macro(sfml_add_library target)
             # include the major version number in Windows shared library names (but not import library names)
             set_target_properties(${target} PROPERTIES DEBUG_POSTFIX -d)
             set_target_properties(${target} PROPERTIES SUFFIX "-${VERSION_MAJOR}${CMAKE_SHARED_LIBRARY_SUFFIX}")
+            configure_file(
+                "${SFML_SOURCE_DIR}/tools/windows/resource.rc.in"
+                "${CMAKE_CURRENT_BINARY_DIR}/${target}.rc"
+            @ONLY)
+            target_sources(${target} PRIVATE "${CMAKE_CURRENT_BINARY_DIR}/${target}.rc")
```

Comment:
Don't forget that target_sources() command is not available with CMake 3.0.2 that SFML supports, needs to be sorted out

Reply:
According to [repology](https://repology.org/metapackage/cmake/versions) the vast majority of
distros support 3.1.3 or greater, so it should be safe enough to bump the minimum version.

Reply:
CMake version has been updated to 3.7.2